Templates/Database/StoredProcedures/StoredProcedures.cst

```diff
@@ -1111,14 +1111,19 @@ public string GetEntityName(bool plural)
 public string GetBySuffix(IList<ColumnSchema> columns)
 {
     System.Text.StringBuilder bySuffix = new System.Text.StringBuilder();
+    int colCnt = 0;
     foreach(var column in columns.AsIndexedEnumerable())
 	{
-	    if (!column.IsFirst) bySuffix.Append("And");
+	    if (column.IsFirst && colCnt >= 1) { 

```

Comment:
Hmm... this seems like it's only going to add an "And" after the first column if there is more than one, but not between each column if there is more than 2.

Reply:
:(  YES! You are right. Fixing it now.

Reply:
Perhaps a simpler solution would be:

`public string GetBySuffix(IList<ColumnSchema> columns)
{
    System.Text.StringBuilder bySuffix = new System.Text.StringBuilder();
    foreach(var column in columns.AsIndexedEnumerable())
	{
	    if (column.IsFirst && !column.IsLast) { 
            bySuffix.Append("And");
        }
	    bySuffix.Append(column.Value.Name);
	}
	return bySuffix.ToString();
}`

Reply:
If there are two or more IndexedEnumerable cols (column.IsFirst & !column.IsLast) would be the solve. If it's the first and last no "And".